blocks/donate/__consent/donate__consent.css

```diff
@@ -0,0 +1,11 @@
+.donate__consent {
+  margin: 30px 0 50px 0;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 18px;
+}
+
+.donate__consent span {
```

Comment:
Can you avoid usage of the tag selector here?

Reply:
I checked and actually it isn't being used so I removed it thanks!